esp/templates/program/modules/admincore/tags.html

```diff
@@ -65,6 +65,8 @@ <h1>{% if program %}Tag Settings for {{ program.niceName }} (ID: {{ program.id }
         <br/>
         <span> {{ field.help_text }} </span>
       {% endif %}
+      <br/>
+      <span style="font-style:italic;">Default: </span>{% if field.field.default != None %}<span>{{ field.field.default }}</span>{% endif %}
```

Comment:
I think it might be cleaner to only include this line where there is a default (that isn't ""), and then maybe add a line to the intro to the page about defaults being listed when they exist. But I also think it's pretty clear that the non-listed defaults would be those where they don't exist based on context.

Reply:
I'm worried people will think the default is whatever the tag is currently set to (which might not be blank) if the default isn't explicitly listed (even if it is blank).